packages/@glimmer/syntax/tests/parser-node-test.ts

```diff
@@ -470,3 +470,15 @@ test("allow {{undefined}} to be passed as a param", function() {
     b.mustache(b.path('foo'), [b.undefined()])
   ]));
 });
+
+test("Handlebars partial should error", function(assert) {
+  assert.throws(() => {
+    parse("{{> foo}}");
+  }, Error(`Handlebars partials are not supported: "{{> foo" on line "1:0"`));
```

Comment:
@locks IMHO `on line "1:0"` looks a bit weird.

- if we say `on line` we should just print the line but not the column
- if we said `at "1:0"` people might not understand that we are talking about a code location

maybe just append `(at 1:0)` at the end?